extensions/README.md

```diff
@@ -16,23 +16,31 @@ include the following script in the `<head>`:
 
 Current list of extended components:
 
-| Component                                     | Description                                                                                 |
-| --------------------------------------------- | ------------------------------------------------------------------------------------------- |
-| [`amp-anim`](amp-anim/amp-anim.md)                     | Runtime-managed animated image, most typically a GIF.                                       |
-| [`amp-audio`](amp-audio/amp-audio.md)                      | Replacement for the HTML5 `audio` tag.                                                      |
-| [`amp-brightcove`](amp-brightcove/amp-brightcove.md)             | Displays a Brightcove Video Cloud or Perform player. |
-| [`amp-carousel`](amp-carousel/amp-carousel.md)                | Generic carousel for displaying multiple similar pieces of content along a horizontal axis. |
-| [`amp-fit-text`](amp-fit-text/amp-fit-text.md)                | Expand or shrink font size to fit the content within the space given.                       |
-| [`amp-font`](amp-font/amp-font.md)                | Trigger and monitor the loading of custom fonts.                       |
-| [`amp-iframe`](amp-iframe/amp-iframe.md)                 | Displays an iframe.                                                                         |
-| [`amp-image-lightbox`](amp-image-lightbox/amp-image-lightbox.md) | Allows for a “image lightbox” or similar experience.                                        |
-| [`amp-instagram`](amp-instagram/amp-instagram.md)           | Displays an instagram embed.                                                                |
-| [`amp-install-serviceworker`](amp-install-serviceworker/amp-install-serviceworker.md)               | Installs a ServiceWorker.
-| [`amp-lightbox`](amp-lightbox/amp-lightbox.md)             | Allows for a “lightbox” or similar experience.                                              |
-| [`amp-list`](amp-list/amp-list.md)             | A dynamic list that can download data and create list items using a template |
-| [`amp-twitter`](amp-twitter/amp-twitter.md)               | Displays a Twitter Tweet.                                                                   |
-| [`amp-vine`](amp-vine/amp-vine.md)               | Displays a Vine simple embed.                                                                   |
-| [`amp-youtube`](amp-youtube/amp-youtube.md)               | Displays a Youtube video.                                                                   |
+| Component | Description |
+| --------- | ----------- |
+| [`amp-access`](amp-access/amp-access.md) | Provides AMP paywall and subscription support.  |
+| [`amp-analytics`](amp-analytics/amp-analytics.md) | Capture analytics data from an AMP document. |
```

Comment:
Small nit-- some descriptions use plural for verb, some don't. Might be nice to make these consistent.